homeassistant/components/wirelesstag/__init__.py

```diff
@@ -95,11 +115,13 @@ def make_notifications(self, binary_sensors, mac):
         _LOGGER.info("Creating configurations for push notifications.")
         configs = []
 
-        bi_url = self.binary_event_callback_url
+        base_url = get_url(self.hass)
```

Comment:
I think that you will prefer internal url ?

Reply:
Yes. Reading https://developers.home-assistant.io/docs/instance_url/ it seems it defaults to the internal url.

Are you recommending some of the parameters to get_url to prefer the internal url?

Reply:
oh right. That's the default, 👍